### ISSUE TITLE
[Lemmy][Bug] Cannot unmarshal string into Go struct field

### ISSUE THREAD

**Issue by tedwardd** [mrusme/neonmodem#23]
Configuring Neon Modem to connect to https://lemmy.sdf.org results in a blank feed. The following error logs are found in `~/.cache/neonmodem.log`

```
{"level":"error","ts":1686705601.7717443,"caller":"aggregator/aggregator.go:77","msg":"aggregator error: json: cannot unmarshal string into Go struct field PostView.posts.subscribed of type bool\n","stacktrace":"github.com/mrusme/neonmodem/aggregator.(*Aggregator).ListPosts\n\t/home/runner/work/neonmodem/neonmodem/aggregator/aggregator.go:77\ngithub.com/mrusme/neonmodem/ui/views/posts.(*Model).refresh.func1\n\t/home/runner/work/neonmodem/neonmodem/ui/views/posts/posts.go:206\ngithub.com/charmbracelet/bubbletea.(*Program).handleCommands.func1.1\n\t/home/runner/go/pkg/mod/github.com/charmbracelet/bubbletea@v0.23.1/tea.go:247"}
{"level":"error","ts":1686705601.7717974,"caller":"posts/posts.go:209","msg":"[json: cannot unmarshal string into Go struct field PostView.posts.subscribed of type bool]","stacktrace":"github.com/mrusme/neonmodem/ui/views/posts.(*Model).refresh.func1\n\t/home/runner/work/neonmodem/neonmodem/ui/views/posts/posts.go:209\ngithub.com/charmbracelet/bubbletea.(*Program).handleCommands.func1.1\n\t/home/runner/go/pkg/mod/github.com/charmbracelet/bubbletea@v0.23.1/tea.go:247"}
{"level":"error","ts":1686705602.1112926,"caller":"aggregator/aggregator.go:77","msg":"aggregator error: json: cannot unmarshal string into Go struct field PostView.posts.subscribed of type bool\n","stacktrace":"github.com/mrusme/neonmodem/aggregator.(*Aggregator).ListPosts\n\t/home/runner/work/neonmodem/neonmodem/aggregator/aggregator.go:77\ngithub.com/mrusme/neonmodem/ui/views/posts.(*Model).refresh.func1\n\t/home/runner/work/neonmodem/neonmodem/ui/views/posts/posts.go:206\ngithub.com/charmbracelet/bubbletea.(*Program).handleCommands.func1.1\n\t/home/runner/go/pkg/mod/github.com/charmbracelet/bubbletea@v0.23.1/tea.go:247"}
{"level":"error","ts":1686705602.1114051,"caller":"posts/posts.go:209","msg":"[json: cannot unmarshal string into Go struct field PostView.posts.subscribed of type bool]","stacktrace":"github.com/mrusme/neonmodem/ui/views/posts.(*Model).refresh.func1\n\t/home/runner/work/neonmodem/neonmodem/ui/views/posts/posts.go:209\ngithub.com/charmbracelet/bubbletea.(*Program).handleCommands.func1.1\n\t/home/runner/go/pkg/mod/github.com/charmbracelet/bubbletea@v0.23.1/tea.go:247"}
```

**Comment by mrusme** [comment]
This might be a side effect of https://github.com/mrusme/neonmodem/pull/19 🤔

**Comment by mrusme** [comment]
I signed up on the instance you mentioned and subscribed to a community. I cannot seem to reproduce this issue:

![screenshot_2023-06-15-092148](https://github.com/mrusme/neonmodem/assets/151967/92f20b24-ee8a-4543-b587-2f8d425395bf)

Do you have any insights on which community might cause this issue?

**Comment by tedwardd** [comment]
Ah! you're correct, the issue seems to be fixed as of v1.0.3 which contains #19 but v1.0.2 did not. Closing.
